osquery/config/config.cpp

```diff
@@ -142,6 +158,11 @@ std::map<std::string, std::vector<std::string> >& Config::getWatchedFiles() {
   return getInstance().cfg_.eventFiles;
 }
 
+std::map<std::string, std::vector<std::string> >& Config::getYARAFiles() {
```

Comment:
Yup, this was a copy/paste of getWatchedFiles(). I'll update this one and give @jedi22 some time to see if he wants to sweep the tree for similar problems.